ext/drreg/drreg.c

```diff
@@ -904,12 +1431,255 @@ drreg_reserve_reg_internal(void *drcontext, instrlist_t *ilist, instr_t *where,
     return DRREG_SUCCESS;
 }
 
+#ifdef SIMD_SUPPORTED
```

Comment:
Add a banner comment (i.e., full line width separator: search for "SPILLING AND RESTORING" in this file for an example) to separate off this section of SIMD code?